src/components/search-panel/SearchPanelForm.tsx

```diff
@@ -134,20 +132,53 @@ const SearchPanelForm = (props: Props) => {
 	};
 
 	return (
-		<>
-			<div className='filter-row'>
-				<div className='filter-row__label'>Start Timestamp</div>
-				<FilterDatetimeInput {...startTimestampInput} />
-			</div>
-			<div className='filter-row'>
-				<div className='filter-row__label'>Time Limit</div>
-				<FilterDatetimeInput {...endTimestampInput} />
+		<div className='search-panel__search-form search-form'>
+			<SearchDatetimeControl
+				form={form}
+				updateForm={updateForm}
+				startTimestampInput={startTimestampInput}
+			/>
+			<AnimatePresence>
+				{!collapsed && (
+					<motion.div
+						initial='collapsed'
+						animate='open'
+						exit='collapsed'
+						variants={{
+							open: { opacity: 1, height: 'auto' },
+							collapsed: { opacity: 0, height: 0 },
+						}}
+						transition={{ duration: 0.3 }}>
+						<div className='search-panel__fields'>
+							<div className='filter-row'>
+								<div className='filter-row__label'>Search for</div>
+								<SearchTypeSwitcher formType={formType} setFormType={setFormType} />
+							</div>
+							<div className='filter-row'>
+								<div className='filter-row__label'>Time Limit</div>
+								<FilterDatetimeInput {...endTimestampInput} />
+							</div>
+							{config.map(rowConfig => (
```

Comment:
I thought we move result count limit somewhere. currently it takes whole row, but it doesnt need it. I dont know maybe move it to same row as `Search for` ? @YourWeebyNeighbor what do you think?

Reply:
Done
![image](https://user-images.githubusercontent.com/34395711/113430111-aa140080-93ea-11eb-975f-d8639fa4c52c.png)